hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/utils/HoodieInputFormatUtils.java

```diff
@@ -438,11 +437,20 @@ public static HoodieMetadataConfig buildMetadataConfig(Configuration conf) {
         if (LOG.isDebugEnabled()) {
           LOG.debug("Hoodie Metadata initialized with completed commit instant as :" + metaClient);
         }
-
         HoodieTimeline timeline = HoodieHiveUtils.getTableTimeline(metaClient.getTableConfig().getTableName(), job, metaClient);
+
```

Comment:
can we combine this into getTableTimeline method? HoodieHiveUtils.getTableTimeline is anyway getting all the config. So i think that provides better abstraction to get relevant timeline.

Reply:
Yes that is great idea. I have made the changes accordingly